### PR TITLE
Make Common 100% again.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: common/src/test/scala/common/validation/ValidationSpec.scala [hunk 1]
```diff
@@ -49,6 +49,21 @@ class ValidationSpec extends FlatSpec with Matchers with Mockito {
     url.getPath should be("/world")
   }
 
+  it should "convert a valid to a successful future" in {
+    val nel: ValidatedNel[String, String] = "ok".valid
+    val actual = nel.toFuture(_ => throw new RuntimeException("Should not be used"))
+    actual.value should be(Option(Success("ok")))
+  }
+
+  it should "convert a invalidNel to a successful future" in {
```
Comment (Horneth) [inline common/src/test/scala/common/validation/ValidationSpec.scala]: `to an unsuccessful future` ?

**Comment by geoffjentry** [comment]
:+1:

File: common/src/test/scala/common/validation/ValidationSpec.scala [hunk 2]
```diff
@@ -49,6 +49,21 @@ class ValidationSpec extends FlatSpec with Matchers with Mockito {
     url.getPath should be("/world")
   }
 
+  it should "convert a valid to a successful future" in {
+    val nel: ValidatedNel[String, String] = "ok".valid
+    val actual = nel.toFuture(_ => throw new RuntimeException("Should not be used"))
```
Comment (cjllanwarne) [inline common/src/test/scala/common/validation/ValidationSpec.scala]: ToL: There's nothing `Runtime` about this `Exception`

File: common/src/test/scala/common/validation/ValidationSpec.scala [hunk 3]
```diff
@@ -49,6 +49,21 @@ class ValidationSpec extends FlatSpec with Matchers with Mockito {
...
+    actual.value should be(Option(Success("ok")))
```
Comment (cjllanwarne) [inline common/src/test/scala/common/validation/ValidationSpec.scala]: That's a `Future`?

**Pull request merged**